metaversefile-api.js

```diff
@@ -1093,7 +1092,7 @@ export default () => {
     app.name = m.name ?? (m.contentId ? m.contentId.match(/([^\/\.]*)$/)[1] : '');
     app.description = m.description ?? '';
     app.appType = m.type ?? '';
-    app.contentId = m.contentId ?? '';
+    // app.contentId = m.contentId ?? '';

```

Comment:
Which cases does this break on?

Reply:
All items which should have PreviewImage were not working cause of this [content Id was wrong] cause of this.

Reply:
Then it sounds like there is a bug in that part of the code, but this isn't the right fix.

Reply:
Perhaps the module is not exposing its `contentId`

Reply:
Maybe.
But we need to stop. We shouldn't fix all existing bugs in this PR.
This definitely should be a separate investigation and PR.